src/components/ChatEntry.js

```diff
@@ -1,22 +1,41 @@
-import React from 'react';
+// import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
 
 const ChatEntry = (props) => {
+  // const [liked, setIsLiked] = useState(false);
+
+  // const likedMesssage = () => {
+  //   setIsLiked(!liked)
+  // }
+
+  const onLikeClick = () =>{
+    props.updateLikes(props.id)
+  }
+
+  const person = (props.sender === 'Vladimir') ? 'chat-entry local' : 'chat-entry remote';
+
+  const heartColor = props.liked ? '❤️' :'🤍'; 
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <section className= {person}>
```

Comment:
Chance for minor refactor: `chat-entry` is in both options, so perhaps something like

```js
  const person = (props.sender === 'Vladimir') ? 'local' : 'emote';
  const heartColor = props.liked ? '❤️' :'🤍'; 
  return (
    <section className= {`chat-entry ${person}`}>
```